frontend/src/app/components/tableWithFilter/table.component.html

```diff
@@ -0,0 +1,79 @@
+<p-table #datatable [value]="data" dataKey="id" [rows]="9" [showCurrentPageReport]="true"
+         [rowsPerPageOptions]="[10,25,50]" styleClass="p-datatable-customers p-datatable-gridlines"
+         [paginator]="true"
+         currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries"
+         [globalFilterFields]="['provinces','activeCases','deaths','hospitalizations']">
+  <ng-template pTemplate="caption">
+    <div class="p-d-flex">
+                <span class="p-input-icon-left p-ml-auto">
+                    <i class="pi pi-search"></i>
+                    <input pInputText type="text" (input)="datatable.filterGlobal($event.target, 'contains')" placeholder="Search keyword" />
+                </span>
+    </div>
+  </ng-template>
+  <ng-template pTemplate="header">
+    <tr>
+      <th>
+        <div class="p-d-flex p-jc-between p-ai-center">
+          Date
```

Comment:
You call the column "Date" but filter the provinces

Reply:
filter geht eh noch nit?!